doc/devdocs/readme.md

```diff
@@ -50,8 +50,23 @@ Various tools used by PowerToys. Includes the Visual Studio 2019 project templat
 
 ### Build Prerequisites
 
+1. install the following:
 - Windows 10 1803 (build 10.0.17134.0) or above to build and run PowerToys.
-- Visual Studio 2019 Community edition or higher, with the 'Desktop Development with C++' component and the Windows 10 SDK version 10.0.18362.0 or higher.
+- Visual Studio 2019 Community edition or higher
+
+2. Run this command in cmd/terminal/ps to install all the workloads and components for VS
+```shell
+"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vs_installer.exe" ^
```

Comment:
⚠ This assumes Visual Studio exists on the box already.

Reply:
yes, see step above.  my thought here was how many people would contribute if it wasn't.  I'll tweak